docs/website-navigation.md

```diff
@@ -140,6 +140,21 @@ site:
 Is populated with page-level metadata and your page's content.
 See [](./frontmatter.md) for many kinds of metadata that configure this section.
 
+### Use the Edit this Page button
+
+If you've added [`github` MyST frontmatter](#table-frontmatter), the MyST themes will display an "Edit this page" link for your page. This link will point to the source file for your page in GitHub, allowing a reader to quickly view the source and make an edit using GitHub's UI.
+
+:::{tip} How to use GitHub's UI to edit a page
+There are two ways to edit a URL in GitHub:
+
+1. Click the pencil icon {kbd}`✏️` to open a lightweight editor on the page.
+2. Replace `.com` with `.dev` in the URL (or, simply press the period button {kbd}`.`). This will open a VSCode editor session you can use to make more extensive edits.
+:::
+
+To override this behavior and set a manual edit URL, use the `edit_url` field in [MyST frontmatter](#table-frontmatter).
+
+To disable the `Edit this page` button, set the value of `edit_url` to `none`.
```

Comment:
Should this be `null` not `none`?

Reply:
I was pulling the text from the `edit_url` frontmatter docs here: https://mystmd.org/guide/frontmatter. But looking at the code I think that `null` is better so I'll switch it to that and @fwkoch should object if he thinks that's wrong!